js/ui/calendar/ui.calendar.base_view.js

```diff
@@ -274,15 +274,16 @@ const BaseView = Widget.inherit({
             return;
         }
 
-        const value = this.option('value');
-        const selectedCell = this._getCellByDate(value);
-
-        if(this._selectedCell) {
-            this._selectedCell.removeClass(CALENDAR_SELECTED_DATE_CLASS);
+        if(this._selectedCells) {
```

Comment:
What about tests? It doesn't clear which scenario you support without them.

Reply:
I think the right way is write tests first according specification and requirement functionality. After that reanimate them step by step.

Reply:
Added tests, while implementation I still wasn't sure what exactly tests I'll need to add